app/css/vendors/ip-camera.css

```diff
@@ -0,0 +1,77 @@
+.camera-view {
+  display: flex;
+  flex-direction: column;
+}
+
+.camera-view__body {
+  flex: 1;
+}
+
+/** HACK until we unify markup for all views, reset footer absolute positioning **/
+.camera-view .footer {
```

Comment:
Footer is currently absolutely positioned, it's just temporal hack so that I can manage view space as flexible as I need (basically I needed middle content to occupy all the space between header and footer). Or maybe I've just missed the original idea behind footer..... Tell me what you think!